src/modal.tsx

```diff
@@ -197,7 +197,7 @@ export class ReactNativeModal extends React.Component<ModalProps, State> {
     showContent: true,
     isVisible: false,
     deviceWidth: Dimensions.get('window').width,
```

Comment:
This should be modified to 'screen' as well.

```suggestion
    deviceWidth: Dimensions.get('screen').width,
```

Scenario: user with virtual android buttons set to hidden with app open.

Steps to reproduce:
- User opens app in landscape mode
- User rotates phone to portrait mode
- User then opens a modal and the spacing meant for when the buttons are visible do not show.